src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -1881,6 +1880,128 @@ class MavlinkStreamVisionPositionEstimate : public MavlinkStream
 	}
 };
 
+class MavlinkStreamOdometry : public MavlinkStream
```

Comment:
It would be better to use the LOCAL_POSITION_NED_COV and ATTITUDE_QUATENRION_COV here (for the flight controller feedback) as you said.

Reply:
I see the point but actually according to the `ODOMETRY` msg spec, we have a `frame_id` that corresponds to the estimated data coming from the FCU. With `LOCAL_POSITION_NED_COV` and `ATTITUDE_QUATENRION_COV` we would continue to have two separate msgs to the odometry parsing, and I think we do want a msg that handles that data. Also `LOCAL_POSITION_NED_COV` has fields that are not really useful to us on that spec.